features/edm4u/edm4u.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Android EDM4U smoke test
+
+  Scenario: Android EDM4U smoke test
+    When On Mobile I relaunch the app
+    And I wait to receive an error
```

Comment:
In theory this _could_ be a source of a flake if the app always send an error when launched.  The app is launched by MR before the scenario starts, so there are 2 requests being generated and in principle they could be received in the same 0.1s interval that MR checks for request with.  To guard against that I'd suggest changing as follows:

```suggestion
    When On Mobile I relaunch the app
    And I wait to receive at least 1 error
```